index.bs

```diff
@@ -107,23 +107,21 @@ In the API, these are represented by the {{AudioSessionType}} enum:
   };
 </pre>
 
+An {{AudioSessionType}} is an <dfn>exclusive type</dfn> if it is {{AudioSessionType/playback}} or {{AudioSessionType/play-and-record}}.
+
 ## Audio session states ## {#audio-session-states}
 
-An [=audio session=] can be in one of following <dfn data-lt="state" for="audio session">state</dfn> , which are represented in the API by the {{AudioSessionState}} enum:
+An [=audio session=] can be in one of the following <dfn data-lt="state" for="audio session">state</dfn> , which are represented in the API by the {{AudioSessionState}} enum:
 
 <dl>
   <dt><dfn for="AudioSessionState" enum-value>active</dfn></dt>
-  <dd>the [=audio session=] is playing sound.</dd>
+  <dd>the [=audio session=] is playing sound/recording microphone.</dd>
```

Comment:
nits : for the consistency, use `or` to replace `/`.

Reply:
Done